### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by joecohens**

### CODE REVIEW

**Comment by Craytor** [comment]
@joecohens Can you please re-base this, as it is two commits behind the master? Thanks!

**Comment by joecohens** [comment]
@Craytor done :)

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To get the latest version of MyWeather Forecast, simply add the following line t
 
 You'll then need to run `composer install` or `composer update` to download it and have the autoloader updated.
 
-If you're using Laravel 5, then you can register our service provider. Open up `config/app.php` and add the following to the `providers` key.
+If you're using Laravel 5, then you can register our service provider. Open up `config/app.php` and add the following to the `providers` array.
```
Comment (GrahamCampbell) [inline README.md:17]: This typo exists in all my laravel packages, lol.

Comment (joecohens) [inline README.md:17]: Lol

Comment (GrahamCampbell) [inline README.md:17]: I'm currently aware of 3 different typos now on all of them. I'll change them all at once whenever I can be arsed, lol.

Comment (joecohens) [inline README.md:17]: Good you notice them :)

Comment (GrahamCampbell) [inline README.md:17]: I've already turned away 2 different pulls to fix them on single repos. I want to do them all at once.

**Pull request merged**